Core/src/org/sleuthkit/autopsy/core/Installer.java

```diff
@@ -207,6 +209,13 @@ private static void loadDynLibraries() {
             } catch (UnsatisfiedLinkError e) {
                 logger.log(Level.SEVERE, "Error loading LIBPQ library, ", e); //NON-NLS
             }
+            
+            try {
```

Comment:
I don't think loadDynLibraries() is the best spot for this. Everything else in this method is being loaded with a call to System.loadLibrary().
I would recommend putting this into it's own method (like we do for JavaFX and Jython) and adding a comment on why this is being done in the installer.